### PR TITLE
Added some width/height constraints

### PR DESCRIPTION
**Pull request by claude**
Why did you make this change?

These constraints were missing from the library and I needed them in my project
so I'm pushing them up.

### CODE REVIEW

File: Constraid/ManageSize.swift [hunk 1]
```diff
@@ -48,4 +48,38 @@ extension UIView {
...
+
+        self.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            NSLayoutConstraint(item: self, attribute: .height, relatedBy: .equal,
+                               toItem: item, attribute: .height, multiplier: multiplier,
+                               constant: constant, priority: priority)
+            ])
+    }
+
+    open func matchWidthToHeight(constant: CGFloat = 0.0,
```
Comment (claude) [inline Constraid/ManageSize.swift]: This was previously named square(), but I didn't like how the name didn't quite follow the rest of library. I'm open to suggestions.

File: Constraid.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -7,6 +7,7 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		51E19A9D1E2A82F900998194 /* ExpandFromWidth.swift in Sources */ = {isa = PBXBuildFile; fileRef = 51E19A9C1E2A82F900998194 /* ExpandFromWidth.swift */; };
```
Comment (cyphactor) [inline Constraid.xcodeproj/project.pbxproj]: Maybe we should rename this file `ExpandFromSize.swift` and have it include expanding from both Width and Height.

File: Constraid/ManageSize.swift (hunk 1 above)
Comment (cyphactor) [inline Constraid/ManageSize.swift]: Hmmm, `makeSquare`, `equalSizeAttributes`, `equalDimensions`, `matchDimensions`

**Comment by cyphactor** [comment]
@claude also noticed that you didn't add any test coverage. I know we don't have great test coverage to start. But, I think we should start requiring test coverage.

File: Constraid/ManageSize.swift (hunk 1 above)
Comment (cyphactor) [inline Constraid/ManageSize.swift]: `equalizeDimensions`

Comment (claude) [inline Constraid/ManageSize.swift]: I think I like makeSquare as it's the easiest to understand

**Comment by claude** [comment]
@cyphactor this PR has been updated to address your comments

**Pull request merged**